js/ripio.js

```diff
@@ -539,6 +541,59 @@ module.exports = class ripio extends Exchange {
         return this.parseTrades (response, market, since, limit);
     }
 
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetPair (params);
+        //
+        //      {
+        //          next: null,
+        //          previous: null,
+        //          results: [
+        //            {
+        //              base: 'BTC',
+        //              base_name: 'Bitcoin',
+        //              quote: 'USDC',
+        //              quote_name: 'USD Coin',
+        //              symbol: 'BTC_USDC',
+        //              fees: [
+        //                {
+        //                  traded_volume: '0.0',
+        //                  maker_fee: '0.0',
+        //                  taker_fee: '0.0',
+        //                  cancellation_fee: '0.0'
+        //                }
+        //              ],
+        //              country: 'ZZ',
+        //              enabled: true,
+        //              priority: '10',
+        //              min_amount: '0.0000100000',
+        //              price_tick: '0.000001',
+        //              min_value: '10',
+        //              limit_price_threshold: '25.00'
+        //            },
+        //            ...
+        //            ]
+        //      }
+        //
```

Comment:
Please, keep uniform comment formatting everywhere, let's use 4 spaces exactly for indentation. **Indent after each opening bracket.**

<img width="636" alt="Screenshot 2022-03-08 at 22 29 44" src="https://user-images.githubusercontent.com/1294454/157311126-d1e462ec-ceb6-4f99-9549-cef624ead2a1.png">

```JavaScript
        //
        //      {
        //          next: null,
        //          previous: null,
        //          results: [
        //            {
        //              base: 'BTC',
        //              base_name: 'Bitcoin',
        //              quote: 'USDC',
        //              quote_name: 'USD Coin',
        //              symbol: 'BTC_USDC',
        //              fees: [
        //                {
        //                  traded_volume: '0.0',
        //                  maker_fee: '0.0',
        //                  taker_fee: '0.0',
        //                  cancellation_fee: '0.0'
        //                }
        //              ],
        //              country: 'ZZ',
        //              enabled: true,
        //              priority: '10',
        //              min_amount: '0.0000100000',
        //              price_tick: '0.000001',
        //              min_value: '10',
        //              limit_price_threshold: '25.00'
        //            },
        //            ...
        //            ]
        //      }
        //
```
↓
```JavaScript
        //
        //     {
        //         next: null,
        //         previous: null,
        //         results: [
        //             {
        //                 base: 'BTC',
        //                 base_name: 'Bitcoin',
        //                 quote: 'USDC',
        //                 quote_name: 'USD Coin',
        //                 symbol: 'BTC_USDC',
        //                 fees: [
        //                     {
        //                         traded_volume: '0.0',
        //                         maker_fee: '0.0',
        //                         taker_fee: '0.0',
        //                         cancellation_fee: '0.0'
        //                     }
        //                 ],
        //                 country: 'ZZ',
        //                 enabled: true,
        //                 priority: '10',
        //                 min_amount: '0.0000100000',
        //                 price_tick: '0.000001',
        //                 min_value: '10',
        //                 limit_price_threshold: '25.00'
        //             },
        //         ]
        //     }
        //
```